test/Interop/Cxx/stdlib/std-module-interface.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=std -source-filename=x -enable-cxx-interop -Xcc -stdlib=libc++ -tools-directory=%llvm_obj_root/bin -module-cache-path %t | %FileCheck %s  -check-prefix=CHECK-STD
+// RUN: %target-swift-ide-test -print-module -module-to-print=std.iosfwd -source-filename=x -enable-cxx-interop -Xcc -stdlib=libc++ -tools-directory=%llvm_obj_root/bin -module-cache-path %t | %FileCheck %s  -check-prefix=CHECK-IOSFWD
+// RUN: %target-swift-ide-test -print-module -module-to-print=std.chrono -source-filename=x -enable-cxx-interop -Xcc -stdlib=libc++ -tools-directory=%llvm_obj_root/bin -module-cache-path %t | %FileCheck %s  -check-prefix=CHECK-CHRONO
+
+// Clang driver on Windows doesn't support -stdlib=libc++
+// XFAIL: OS=windows-msvc
```

Comment:
Would be good to see if we can import MSVC's STL. (And if so, add a test for it.)

Reply:
We can't currently import it: the problem with MSVC's stdlib is that it doesn't have a module map. This should be possible to workaround by manually adding the `std` definition here: https://github.com/apple/swift/blob/main/stdlib/public/Platform/visualc.modulemap, but I'm not sure that's the best approach.